ios/RNGoogleMobileAds/RNGoogleMobileAdsBannerComponent.m

```diff
@@ -95,6 +95,14 @@ - (void)requestAd {
   [self addSubview:_banner];
   _banner.adUnitID = _unitId;
   [self setRequested:YES];
+  _banner.paidEventHandler = ^(GADAdValue *_Nonnull value) {
```

Comment:
This should also be reflected in the new architecture: RNGoogleMobileAdsBannerView.mm

Reply:
Really glad I pinged you @dylancom ! You've done all the new arch work and I've got zero new arch experience so far

Reply:
Ok, missed that as I assumed that both new and old arch banners should be sharing the same core code responsible for interaction with AdMob SDK. I believe, I took care of it in the commits below, it seems to work for me fine with the new arch now.

Reply:
Hey @dylancom , @mikehardy , just double-checking that you have seen that I had made necessary updates.